YogaKit/Source/YGLayout.h

```diff
@@ -10,9 +10,25 @@
 #import <UIKit/UIKit.h>
 #import <yoga/YGEnums.h>
 
+@class YGLayout;
+
+@protocol YGLayoutEntity <NSObject>
+@property (nonatomic, readonly, strong) YGLayout *yoga;
```

Comment:
I could but then implementing the protocol would be a bit more cumbersome. Is there a specific reason for preferring method signatures over properties?